Windows build error using msvc 142

```
D:\include\ftxui/screen/color.hpp(22): error C2059: syntax error: "("
D:\include\ftxui/screen/color.hpp(22): error C2091: function return function
D:\include\ftxui/screen/color.hpp(22): error C2143: syntax error: missing "," (before "|")
D:\include\ftxui/screen/color.hpp(22): error C2535: "ftxui::Color::Color(void)": Member function has been defined or declared
D:\include\ftxui/screen/color.hpp(17): note: see the declaration of "ftxui::Color::Color"
D:\include\ftxui/screen/color.hpp(22): error C2059: syntax error: "|"
D:\include\ftxui/screen/color.hpp(22): error C2059: syntax error: ")"
D:\include\ftxui/screen/color.hpp(22): error C2238: Unexpected mark is located before ";"
```

--- a/include/ftxui/screen/color.hpp
+++ b/include/ftxui/screen/color.hpp
@@ -4,6 +4,12 @@
 #include <cstdint>
 #include <string>
 
+#ifdef RGB
+// Workaround for wingdi.h (via Windows.h) defining macros that break things.
+// https://docs.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-rgb
+#undef RGB
+#endif
+
 namespace ftxui {
 
 /// @brief A class representing terminal colors.

